bin/setup

```diff
@@ -28,6 +28,13 @@ if ! command -v psql > /dev/null; then
   exit -1
 fi
 
+if ! command -v elasticsearch > /dev/null; then
+  printf "${CLEAR_LINE}${RED}  You must install elasticsearch before setup can continue${NO_COLOR}\n"
+  printf "    You should:\n"
+  printf "      $ brew install elasticsearch\n"
```

Comment:
Good point! Ideally we want to test and develop on 2.x as we use 2.1 in production - 2.1 used to be the default version via Homebrew, but it appears to be 5 now. 

Judging by `brew search`, Homebrew only offers 2.4 as a managed version currently, so maybe `brew install elasticsearch@2.4` would be better here.